reference/core/crds.md

```diff
@@ -37,7 +37,7 @@ Note that the `namespace` must be declared in the `metadata`, but if needed, `am
 
 ## Supported CRDs
 
-The full set of CRDs supported by the Ambassador Edge Stack:
+You can run `kubectl get ambassador-crds` The full set of CRDs supported by the Ambassador Edge Stack:
```

Comment:
Did you mean to say "You can run 'kubectl get ambassador-crds' to see the ..." here?